### PR TITLE
Reset textbox text after creating task

### PR DESCRIPTION
**Pull request by BachToTheFuture**
This should fix issue #4, where the old name of the previously created tasks won't delete.

### CODE REVIEW

File: src/NewTaskView.cpp [hunk 1]
```diff
@@ -45,11 +45,12 @@ NewTaskView::MessageReceived(BMessage* message)
 	case BUTTON_NEW_TASK_OK:
 	{
 		BMessage* Temp;
-		if (m_TextControl->Text() == "")
+		if (m_TextControl->Text()[0] == '\0')
```
Comment (humdingerb) [inline src/NewTaskView.cpp:48]: Why the change from `if (m_TextControl->Text() == "")`?
(Another possibilty would be `if (m_TextControl->TextLength() == 0)` btw.)

**Comment by humdingerb** [comment]
There seem to be changes from your (already merged) pr #28 in here...

File: src/NewTaskView.cpp (hunk 1 above)
Comment (BachToTheFuture) [reply src/NewTaskView.cpp:48]: @humdingerb The `if (m_TextControl->Text() == "")` doesn't check for an empty string correctly, so if I leave it with `if (m_TextControl->Text() == "")`, a new task with no name is added when I leave the textbox blank and press OK. The changes should fix this bug.

Edit: Additional info

Comment (BachToTheFuture) [reply src/NewTaskView.cpp:48]: @humdingerb I tried `if (m_TextControl->TextLength() == 0)`, but it does the same thing as `if (m_TextControl->Text() == "")`. I think I'll leave it as it is 😄

**Comment by BachToTheFuture** [comment]
@humdingerb Please help me how to use git correctly :o

**Comment by humdingerb** [comment]
resolved with #33
